lib/ssh/src/ssh_sftpd.erl

```diff
@@ -60,14 +60,15 @@
 %%====================================================================
 -spec subsystem_spec(Options) -> Spec when
       Options :: [ {cwd, string()} |
-                   {file_handler, CallbackModule::string()} |
+                   {file_handler, CbMod | {CbMod, FileState}} |
                    {max_files, integer()} |
                    {root, string()} |
                    {sftpd_vsn, integer()}
                  ],
       Spec :: {Name, {CbMod,Options}},
       Name :: string(),
-      CbMod :: atom() .
+      CbMod :: atom(),
+      FileState :: ssh_sftpd_file_api:state().
```

Comment:
It seems best to just do the minimum error fix, that is instead of ssh_sftpd_file_api:state() just use term().  We are also a bit reluctant to introduce too many types.